.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 language: python
 
 python:
-- 3.5.2
+- 3.5.3
```

Comment:
3.5.2 was here to emphasize the difference between 3.5.2 and 3.5.3
Let's just drop redundant travis job. I could help to reduce travis build time.

Reply:
@asvetlov removed.